dart/test/benchmark_test.dart

```diff
@@ -2,6 +2,9 @@ import 'package:open_location_code/open_location_code.dart' as olc;
 import 'package:test/test.dart';
```

Comment:
_This is an automated bot comment from the TravisCI tests_  
**File has `dartanalyzer` errors that must be fixed**:
Analyzing test/benchmark_test.dart...  
  hint • The operator x ~/ y is more efficient than (x / y).toInt() at test/benchmark_test.dart:6:11 • division_optimization  
1 hint found.